src/components/query/QueryDirective.js

```diff
@@ -49,12 +53,16 @@ goog.require('ga_storage_service');
           return;
         }
         if (!paramsByLayer[filter.layer.bodId]) {
+          var params = {};
+          var timeInstant = getTimeInstantParam(filter.layer);
+          if (timeInstant) {
```

Comment:
@gjn this PR is mainly because of that. We would send the time instant params even if the layer was not time enabled.